test/spec/modules/amxBidAdapter_spec.js

```diff
@@ -0,0 +1,384 @@
+import * as utils from 'src/utils.js';
+import { config } from 'src/config.js';
+import { expect } from 'chai';
+import { newBidder } from 'src/adapters/bidderFactory.js';
+import { spec } from 'modules/amxBidAdapter.js';
+import { BANNER, VIDEO } from 'src/mediaTypes';
+import { formatQS } from 'src/utils';
```

Comment:
You don't need to import this function from utils, your first line already imports everything from utils:

```
import * as utils from 'src/utils.js';
```

You could remove this line, and in your tests switch from `formatQS()` to `utils.formatQS()`.